### ISSUE TITLE
No stream detected on the FaceBook side

### ISSUE THREAD

**Issue by lakano** [digitallysavvy/agora-web-broadcast-demo#1]
Hello,
Thanks for your great demo sample! 
I've updated my AgoraAppId key, and I try to stream to FaceBook RTMPS, but there is no bug in the JS console and FaceBook doesn't receive anything.

I've tried with or without publishing before (url set to empty, current url, and facebook RTMPS url). 
Could you please confirm me this still working on your side ?

**Comment by digitallysavvy** [comment]
Currently Agora's platform does not support pushing to RTMPS only RTMP so pushing to Facebook will not work. I will ask internally what is the status for supporting RTMP and I'll post a comment once I have an update.

**Comment by lakano** [comment]
Thanks for your feedback @digitallysavvy 
I've created a proxy RTMTP => RTMPS on my own server (with stunnel), this works with ffmpeg correctly, but still not works from your code or my code to FaceBook.
BTW, Youtube doesn't force to RTMPS and doesn't works too.
On my code, the exact error is APPCENTER_CODE_400 with an object said « appid_invalid » ... it's strange because my APPID is correctly set... 

UPDATE: Strangely, my APP wasn't enabled in the RTMP Duration function. 
Now it's stream correctly to Youtube, but not to FaceBook. My proxy RTMP to RTMPS is refused per FaceBook (rtmp-handshake error), and if I set directly the RTMPS URL to Agora.io, it's return an URL parse error... So I suppose Agora doesn't support RTMPS.
